platform/android/java/editor/src/main/java/org/godotengine/editor/GodotEditor.kt

```diff
@@ -118,6 +118,7 @@ open class GodotEditor : GodotActivity() {
 			godotFragment?.godot?.renderView?.inputHandler?.apply {
 				enableLongPress(longPressEnabled)
 				enablePanningAndScalingGestures(panScaleEnabled)
+				setRotaryInputAxis(rotaryInputAxis)
```

Comment:
You can also revert this change for the `GodotEditor`.

Reply:
fixed